SQRLUtilsLib/SQRLCPSServer.cs

```diff
@@ -113,7 +137,7 @@ private void Listen()
         {
             try
             {
-                var prefix = "http://localhost:" + this.Port.ToString() + "/";
+                var prefix = "http://localhost:" + Port.ToString() + "/";
```

Comment:
We could use string interpolation here a little cleaner
var prefix = $"http://localhost:{Port}/";

Reply:
PS: That's probably my original code LoL but while we  are here... 😉

Reply:
Yep, that would indeed look much cleaner!
Fixing it now!

Reply:
Fixed!